src/views/AddItem/AddItem.jsx

```diff
@@ -19,7 +32,48 @@ export default function AddItem({ token }) {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    addItem();
+    const checkForErrors = validate();
+    if (checkForErrors) {
+      setError('Item already in list!');
+      setListItem({
+        name: '',
+        frequency: '7',
+      });
+      setTimeout(() => {
+        setError('');
+      }, 3000);
+      return;
+    } else {
+      addItem();
+      setListItem({
+        name: '',
+        frequency: '7',
+      });
+      setError('Item added successfully!');
+      setTimeout(() => {
+        setError('');
+      }, 3000);
```

Comment:
🔍 Great work on the feature, and thank you for doing the additional confirmation message! This works great in my testing. One thing: there are some accessibility risks to getting rid of a notification on a timeout. What if a user hasn't read it in time? Generally, it's best to let the user have as much time as they need before the notification goes away. In this app, that could mean:

- letting the notification persist until the user attempts to add another item, at which time the existing notification is replaced, or
- adding a button that the user can click in order to dismiss the message at will, or
- some combination of these two

Right now, the first implementation would be the easiest. You'd only need to delete a little bit of code, if I understand the current code correctly.